LoopKit/DeviceManager/DeviceManager.swift

```diff
@@ -53,4 +56,9 @@ public extension DeviceManager {
     var localizedTitle: String {
         return type(of: self).localizedTitle
     }
+    
+    var deviceManagerIdentifier: String {
```

Comment:
Could also just be `managerIdentifier`, as the context (class vs instance) here doesn't matter; it is really the same thing, and the same meaning.

Reply:
heh that's what I had in the first place...I'll change it back /shrug